lisp/plugins/gst_backend/settings/alsa_sink.py

```diff
@@ -76,10 +73,12 @@ def enableCheck(self, enabled):
         self.setGroupEnabled(self.deviceGroup, enabled)
 
     def loadSettings(self, settings):
-        device = settings.get("device", "default")
+        device = settings.get("device", "")
+        if not device:
+            device = get_plugin('GstBackend').Config.get('alsa_device', AlsaSink.FALLBACK_DEFAULT_DEVICE)
```

Comment:
Same as before, we can use `GstBackend` directly, and probably just pass the config value as default?

```suggestion
        device = settings.get("device", GstBackend.Config.get('alsa_device', AlsaSink.FALLBACK_DEFAULT_DEVICE))
```